packages/freezed/lib/src/tools/type.dart

```diff
@@ -73,9 +78,41 @@ String resolveFullTypeStringFrom(
     }
   }
 
+  // The parameter is a Interface with a Type Argument that is not yet generated
+  // In this case analyzer would set its type to InvalidType
+  //
+  // For example for:
+  // List<ToBeGenerated> values,
+  //
+  // it would generate:  List<InvalidType>
+  // instead of          List<dynamic>
+  //
+  // This a regression in analyzer 5.13.0
+  if (type is InterfaceType &&
+      type.typeArguments.contains(InvalidTypeImpl.instance)) {
+    var modified = type;
+    modified.typeArguments
+      ..replaceWhere(
+        (t) => t is InvalidType,
+        DynamicTypeImpl.instance,
```

Comment:
Use TypeSystem/TypeProvider. You should be able to obtain a dynamicType more officially

Reply:
How would I get access to the TypeProvider?
Might need an example, do you use in riverpod_generator?

Reply:
From LibraryElement, which is accessible from any element.